testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/display/firsttime/FirstTimeLaunch.kt

```diff
@@ -0,0 +1,9 @@
+package com.rakuten.tech.mobile.testapp.ui.display.firsttime
+
+import androidx.annotation.Keep
+
+@Keep
+data class FirstTimeLaunch(
```

Comment:
I think you can just simply use the SharedPreference with `key: String, value: Boolean`.

Reply:
Yeah, Thanks for the feedback, I have updated this part.